src/Components/BookDetail/Reviews/ReviewSummary/ReviewSummary.jsx

```diff
@@ -0,0 +1,27 @@
+function ReviewSummary ({reviews}) {
+
+    let reviewCount = {reviews}.reviews.length
+    let reviewArray = {reviews}.reviews
+    let reviewTally = 0
+    let reviewAverage = 0
+
+    reviewArray.forEach(review => {
+        reviewTally += review.rating
+    })
+
+    // Handle 0 review scenario, to fix NaN dividing by 0
+    if(reviewCount <= 0)
+        {
+        reviewAverage = 0    
+        }
+    else
+        {
+        reviewAverage = Math.round((reviewTally / reviewCount) * 10) / 10            
+        }
```

Comment:
Check indenting/formatting

Opening { should be ont he same line as the if/else

Reply:
Fixed - (been Whitesmith for a while, still slipping back into it instead of K&R style at times!)